index.bs

```diff
@@ -1564,16 +1564,18 @@ partial interface MLGraphBuilder {
     The behavior of this operation when the input tensor is 4-D of the {{MLInputOperandLayout/"nchw"}} layout can be [EMULATED]
     </summary>
     <pre highlight="js">
-    const shape = [1, input.shape()[options.axis], 1, 1];
-    return builder.add(
-      builder.mul(
-        builder.reshape(options.scale, shape),
-        builder.div(
-          builder.sub(input, builder.reshape(mean, shape)),
-          builder.sqrt(builder.add(
-            builder.reshape(variance, shape),
-            builder.constant(input.dataType, options.epsilon))))),
-      builder.reshape(options.bias, shape));
+    function batchNormalization(builder, input, mean, variance, options) {
+      const shape = [1, input.shape()[options.axis], 1, 1];
+      return builder.add(
+        builder.mul(
+          builder.reshape(options.scale, shape),
+          builder.div(
+            builder.sub(input, builder.reshape(mean, shape)),
+            builder.sqrt(builder.add(
+              builder.reshape(variance, shape),
+              builder.constant(input.dataType, options.epsilon))))),
```

Comment:
This was a previous issue, `dataType` is a method rather than an attribute. Same for the other occurrences. 
```suggestion
              builder.constant(input.dataType(), options.epsilon))))),
```

Reply:
Done in 46442ad